docs/01-Using-Fleet/03-REST-API.md

```diff
@@ -5592,3 +5592,29 @@ _Available in Fleet Premium_
   }
 }
 ```
+
+### Count software
+
+`GET /api/v1/fleet/software`
+
+#### Parameters
```

Comment:
This is missing some parameters, maybe just explain it has the same parameters as software listing.

Reply:
https://github.com/fleetdm/fleet/pull/3105/files#r758285427